<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneWindows</class>
 <widget class="QWidget" name="SceneWindows">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SceneWindows</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>471</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>691</width>
     <height>471</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>691</width>
     <height>471</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>432</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>432</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;场景A&lt;/p&gt;&lt;p&gt;介绍：&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;1&lt;/p&gt;&lt;p&gt;&lt;img src=&quot;:/MainWindows/D:/图片/test.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>432</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>432</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;场景B&lt;br/&gt;介绍：&lt;/p&gt;&lt;p&gt;。。。。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>432</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>432</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;场景C&lt;/p&gt;&lt;p&gt;介绍：&lt;/p&gt;&lt;p&gt;......&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>223</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>223</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindows.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>signal1()</signal>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot3()</slot>
 </slots>
</ui>
